### PR TITLE
SplitItemBuilder should reference menu styles

### PR DESCRIPTION
**Pull request by jtreminio**
Related to #205

### CODE REVIEW

File: src/Builder/SplitItemBuilder.php [hunk 1]
```diff
@@ -171,12 +158,12 @@ public function build() : SplitItem
 
     public function getCheckboxStyle() : CheckboxStyle
     {
-        return $this->checkboxStyle;
+        return $this->menu->getCheckboxStyle();
     }
 
     public function setCheckboxStyle(CheckboxStyle $style) : self
```
Comment (AydinHassan) [inline src/Builder/SplitItemBuilder.php]: Can we drop the setters?

Comment (AydinHassan) [reply src/Builder/SplitItemBuilder.php]: Sorry and the `modify*` methods.

Comment (jtreminio) [reply src/Builder/SplitItemBuilder.php]: `SplitItemBuilder::setCheckboxStyle()` is called from `CliMenuBuilder::addSplitItem()` so the split item can inherit the parent styles.

**Comment by AydinHassan** [comment]
OK it works but I can't wrap my head around it. It's getting late here now anyway, need to sleep.

**Pull request merged**